src/databricks/sql/__init__.py

```diff
@@ -44,7 +44,6 @@ def TimestampFromTicks(ticks):
     return Timestamp(*time.localtime(ticks)[:6])
 
 
-def connect(server_hostname, http_path, access_token, **kwargs):
+def connect(server_hostname, http_path, experimental_oauth_persistence=None, **kwargs):
     from .client import Connection
-
-    return Connection(server_hostname, http_path, access_token, **kwargs)
+    return Connection(server_hostname, http_path, experimental_oauth_persistence, **kwargs)
```

Comment:
This file can be reverted; no need to call out `experimental_oauth_persistence` here as it is carried forward by `**kwargs` just like all the other optional keyword arguments.

Reply:
thanks for the comment. addressed.